lib/git.zsh

```diff
@@ -36,7 +36,10 @@ git_remote_status() {
         ahead=$(command git rev-list ${hook_com[branch]}@{upstream}..HEAD 2>/dev/null | wc -l)
         behind=$(command git rev-list HEAD..${hook_com[branch]}@{upstream} 2>/dev/null | wc -l)
 
-        if [ $ahead -gt 0 ] && [ $behind -eq 0 ]
+        if [ $ahead -eq 0 ] && [ $behind -eq 0 ]
+        then
+          echo "$ZSH_THEME_GIT_PROMPT_EQUAL_REMOTE"
```

Comment:
Unfortunately this change did break existing themes. I'm using the strug theme and the echo output causes some wired line break. Changing the echo output to a variable assignment fixed it for me. ```git_remote_status="$ZSH_THEME_GIT_PROMPT_EQUAL_REMOTE"```